### ISSUE TITLE
Proposition to solve the "title" problem on Youtube

### ISSUE THREAD

**Issue by dinh** [osmoscraft/osmosmemo#8]
I know that is is not the proper way to do. Sorry in advance...

```
  function getPageTitle() {
    let title;
	
    if (document.location.href.includes("youtube.com/channel")) {
      document.querySelector("#text-container").innerText
    } else if (document.location.href.includes("youtube.com")) {
      title = document.querySelector("#container > h1").innerText;
    } else {
      title = document.querySelector(`meta[property="og:title"]`)?.getAttribute("content")?.trim();
    }

    if (!title) {
      title = document.querySelector(`meta[name="twitter:title"]`)?.getAttribute("content")?.trim();
    }
    if (!title) {
      title = document.querySelector("title")?.innerText?.trim();
    }
    if (!title) {
      title = document.querySelector("h1")?.innerText?.trim();
    }
    if (!title) {
      title = "";
    }
    return title;
  }
```

**Comment by chuanqisun** [comment]
Quick update, I plan to implement the special case handler as suggested by @dinh. Sorry it took me so long to triage this.

**Comment by chuanqisun** [comment]
Fixed with #12 and submitted to extension stores. There will be [some delay](https://github.com/osmoscraft/osmosmemo#how-long-does-it-take-for-a-new-release-to-reach-my-browser) before each browser accepts it.
